<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="draw_wdg" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>840</width>
      <height>720</height>
     </rect>
    </property>
    <widget class="QGraphicsView" name="graphicsView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>840</width>
       <height>720</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="sceneRect">
      <rectf>
       <x>0.000000000000000</x>
       <y>0.000000000000000</y>
       <width>840.000000000000000</width>
       <height>720.000000000000000</height>
      </rectf>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="actions_frm">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>0</y>
      <width>240</width>
      <height>640</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>-1</y>
       <width>241</width>
       <height>641</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="rot_frm_2">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <widget class="QLabel" name="rot_lbl_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>240</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
           <strikeout>false</strikeout>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="text">
          <string>Rotate</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="angle_lbl_2">
         <property name="geometry">
          <rect>
           <x>55</x>
           <y>40</y>
           <width>180</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Rotation  angle = </string>
         </property>
        </widget>
        <widget class="QWidget" name="buttons_rot_wdg_4" native="true">
         <property name="geometry">
          <rect>
           <x>5</x>
           <y>95</y>
           <width>230</width>
           <height>111</height>
          </rect>
         </property>
         <widget class="QWidget" name="gridLayoutWidget_4">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>242</width>
            <height>111</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <property name="spacing">
            <number>0</number>
           </property>
           <item row="2" column="0">
            <widget class="QPushButton" name="nrot_ox_btn">
             <property name="text">
              <string>-</string>
             </property>
             <property name="autoRepeat">
              <bool>true</bool>
             </property>
             <property name="autoRepeatInterval">
              <number>40</number>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="rot_oy_lbl">
             <property name="text">
              <string>OY</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="rot_ox_lbl">
             <property name="text">
              <string>OX</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QPushButton" name="prot_ox_btn">
             <property name="text">
              <string>+</string>
             </property>
             <property name="autoRepeat">
              <bool>true</bool>
             </property>
             <property name="autoRepeatInterval">
              <number>40</number>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="rot_oz_lbl">
             <property name="text">
              <string>OZ</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="prot_oy_btn">
             <property name="text">
              <string>+</string>
             </property>
             <property name="autoRepeat">
              <bool>true</bool>
             </property>
             <property name="autoRepeatInterval">
              <number>40</number>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="prot_oz_btn">
             <property name="text">
              <string>+</string>
             </property>
             <property name="autoRepeat">
              <bool>true</bool>
             </property>
             <property name="autoRepeatInterval">
              <number>40</number>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="nrot_oy_btn">
             <property name="text">
              <string>-</string>
             </property>
             <property name="autoRepeat">
              <bool>true</bool>
             </property>
             <property name="autoRepeatInterval">
              <number>40</number>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QPushButton" name="nrot_oz_btn">
             <property name="text">
              <string>-</string>
             </property>
             <property name="autoRepeat">
              <bool>true</bool>
             </property>
             <property name="autoRepeatInterval">
              <number>40</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QDial" name="rot_dial">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>25</y>
           <width>50</width>
           <height>50</height>
          </rect>
         </property>
         <property name="minimum">
          <number>5</number>
         </property>
         <property name="maximum">
          <number>180</number>
         </property>
         <property name="singleStep">
          <number>5</number>
         </property>
         <property name="pageStep">
          <number>50</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="wrapping">
          <bool>true</bool>
         </property>
         <property name="notchesVisible">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="move_frm_2">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <widget class="QLabel" name="scl_lbl">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>240</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
           <strikeout>false</strikeout>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="text">
          <string>Scale</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QWidget" name="buttons_rot_wdg_5" native="true">
         <property name="geometry">
          <rect>
           <x>5</x>
           <y>95</y>
           <width>230</width>
           <height>111</height>
          </rect>
         </property>
         <widget class="QWidget" name="gridLayoutWidget_5">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>242</width>
            <height>111</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <property name="spacing">
            <number>0</number>
           </property>
           <item row="2" column="0">
            <widget class="QPushButton" name="nscl_ox_btn">
             <property name="text">
              <string>-</string>
             </property>
             <property name="autoRepeat">
              <bool>true</bool>
             </property>
             <property name="autoRepeatInterval">
              <number>40</number>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="scl_oy_lbl">
             <property name="text">
              <string>OY</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="scl_ox_lbl">
             <property name="text">
              <string>OX</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QPushButton" name="pscl_ox_btn">
             <property name="text">
              <string>+</string>
             </property>
             <property name="autoRepeat">
              <bool>true</bool>
             </property>
             <property name="autoRepeatInterval">
              <number>40</number>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="scl_oz_lbl">
             <property name="text">
              <string>OZ</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="pscl_oy_btn">
             <property name="text">
              <string>+</string>
             </property>
             <property name="autoRepeat">
              <bool>true</bool>
             </property>
             <property name="autoRepeatInterval">
              <number>40</number>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="pscl_oz_btn">
             <property name="text">
              <string>+</string>
             </property>
             <property name="autoRepeat">
              <bool>true</bool>
             </property>
             <property name="autoRepeatInterval">
              <number>40</number>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="nscl_oy_btn">
             <property name="text">
              <string>-</string>
             </property>
             <property name="autoRepeat">
              <bool>true</bool>
             </property>
             <property name="autoRepeatInterval">
              <number>40</number>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QPushButton" name="nscl_oz_btn">
             <property name="text">
              <string>-</string>
             </property>
             <property name="autoRepeat">
              <bool>true</bool>
             </property>
             <property name="autoRepeatInterval">
              <number>40</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QLabel" name="coefficient_lbl_2">
         <property name="geometry">
          <rect>
           <x>55</x>
           <y>40</y>
           <width>180</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Scale coefficient = </string>
         </property>
        </widget>
        <widget class="QDial" name="scl_dial">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>25</y>
           <width>50</width>
           <height>50</height>
          </rect>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>50</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="pageStep">
          <number>5</number>
         </property>
         <property name="value">
          <number>49</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="wrapping">
          <bool>true</bool>
         </property>
         <property name="notchesVisible">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="scale_frm_2">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <widget class="QLabel" name="move_lbl_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>240</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
           <strikeout>false</strikeout>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="text">
          <string>Move</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QWidget" name="buttons_rot_wdg_6" native="true">
         <property name="geometry">
          <rect>
           <x>5</x>
           <y>95</y>
           <width>230</width>
           <height>111</height>
          </rect>
         </property>
         <widget class="QWidget" name="gridLayoutWidget_6">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>242</width>
            <height>111</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <property name="spacing">
            <number>0</number>
           </property>
           <item row="2" column="0">
            <widget class="QPushButton" name="nmv_ox_btn">
             <property name="text">
              <string>-</string>
             </property>
             <property name="autoRepeat">
              <bool>true</bool>
             </property>
             <property name="autoRepeatInterval">
              <number>40</number>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="move_oy_lbl">
             <property name="text">
              <string>OY</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="move_ox_lbl">
             <property name="text">
              <string>OX</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QPushButton" name="pmv_ox_btn">
             <property name="text">
              <string>+</string>
             </property>
             <property name="autoRepeat">
              <bool>true</bool>
             </property>
             <property name="autoRepeatInterval">
              <number>40</number>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="move_oz_lbl">
             <property name="text">
              <string>OZ</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="pmv_oy_btn">
             <property name="text">
              <string>+</string>
             </property>
             <property name="autoRepeat">
              <bool>true</bool>
             </property>
             <property name="autoRepeatInterval">
              <number>40</number>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="pmv_oz_btn">
             <property name="text">
              <string>+</string>
             </property>
             <property name="autoRepeat">
              <bool>true</bool>
             </property>
             <property name="autoRepeatInterval">
              <number>40</number>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="nmv_oy_btn">
             <property name="text">
              <string>-</string>
             </property>
             <property name="autoRepeat">
              <bool>true</bool>
             </property>
             <property name="autoRepeatInterval">
              <number>40</number>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QPushButton" name="nmv_oz_btn">
             <property name="text">
              <string>-</string>
             </property>
             <property name="autoRepeat">
              <bool>true</bool>
             </property>
             <property name="autoRepeatInterval">
              <number>40</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QDial" name="move_dial">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>25</y>
           <width>50</width>
           <height>50</height>
          </rect>
         </property>
         <property name="minimum">
          <number>10</number>
         </property>
         <property name="maximum">
          <number>200</number>
         </property>
         <property name="singleStep">
          <number>10</number>
         </property>
         <property name="pageStep">
          <number>50</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
         <property name="sliderPosition">
          <number>10</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="wrapping">
          <bool>true</bool>
         </property>
         <property name="notchesVisible">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLabel" name="div_lbl_2">
         <property name="geometry">
          <rect>
           <x>55</x>
           <y>40</y>
           <width>180</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Coordinates step = </string>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="show_btn">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>650</y>
      <width>100</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Load model</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
